test/remote/sms_tests.js

```diff
@@ -38,3 +39,73 @@ describe('remote sms', () => {
     return TestServer.stop(server)
   })
 })
+
+describe('remote sms (mocked nexmo)', () => {
+  let server
+
+  before(() => {
+    config.sms.enabled = true
+    // POST /sms spends actual money unless the SMS provider is mocked
+    config.sms.useMock = true
+
+    return TestServer.start(config)
+      .then(result => {
+        server = result
+      })
+  })
+
+  it('POST /sms success', () => {
```

Comment:
There doesn't seem to be a test for mock `GET /status`, I think we should add it.